src/Service/ApiServiceInterface.php

```diff
@@ -79,4 +82,10 @@ public function getIssuesDataForProject(string $projectId): array;
      * @return array<AccountData>
      */
     public function getAllAccountData(): array;
+
+    public function getAllProjectsV2(): SprintReportProjects;
```

Comment:
These 3 functions are already added to ProjectTrackerInterface. Remove them from ApiServiceInterface.

Reply:
Which three?